cea/technologies/solar/photovoltaic_thermal.py

```diff
@@ -653,7 +653,11 @@ def calc_Cinv_PVT(PVT_peak_W, locator, technology=0):
             PVT_peak_W = PVT_cost_data['cap_min'].values[0]
         PVT_cost_data = PVT_cost_data[
             (PVT_cost_data['cap_min'] <= PVT_peak_W) & (PVT_cost_data['cap_max'] > PVT_peak_W)]
-        Inv_a = PVT_cost_data.iloc[0]['a']
+        try:
+            Inv_a = PVT_cost_data.iloc[0]['a']
+        except Exception as error:
+            print('Caught this error: ' + repr(error))
+            print(PVT_cost_data, PVT_peak_W)
```

Comment:
I think we should raise an Exception when `PVT_cost_data` is empty, instead of ignoring the error, since `Inv_a` will not be defined if we catch the exception like this.

Reply:
fixed